doc/INSTALL-WINDOWS

```diff
@@ -39,8 +39,7 @@ system.
       cd c:\Users\YourName\Desktop\john\src
 
     - Build
-      ./configure && make -s clean && make -sj4
-      make windows
+      configure && make -s clean && make -sj4
```

Comment:
`.` is in PATH in default cygwin?

Reply:
Since we require running this in Cygwin shell, I think it's better to keep the "./" - it won't hurt, but it and the habit might help on other occasions.